### PR TITLE
Temporarily hide file attachment functionality in the UI

### PR DESCRIPTION
**Pull request by nmajor25**
Shortcut Story ID: [sc-33229]

Areas the UI where this needs to be hidden:

- Attachments section of the action sidebar.
- Paperclip icon added to any action list item that has an attachment (worklists, archive, patient dashboard, flow pages).

### CODE REVIEW

**Comment by nmajor25** [comment]
This PR isn't meant to be merged into develop. I'll create a release directly off this branch once approved.

File: src/js/apps/patients/sidebar/action-sidebar_app.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ export default App.extend({
 
     this.showActivity();
     this.showNewCommentForm();
-    this.showAttachments();
+    // this.showAttachments();
```
Comment (nmajor25) [inline src/js/apps/patients/sidebar/action-sidebar_app.js:44]: Hides the attachments section in the action sidebar.

File: src/js/views/patients/patient/archive/archive_views.js [hunk 2]
```diff
@@ -81,7 +81,8 @@ const ActionItemView = View.extend({
   templateContext() {
     return {
       icon: this.model.hasOutreach() ? 'share-from-square' : 'file-lines',
-      hasAttachments: this.model.hasAttachments(),
+      // hasAttachments: this.model.hasAttachments(),
+      hasAttachments: false,
```
Comment (nmajor25) [inline src/js/views/patients/patient/archive/archive_views.js:85]: Each of these hides the paperclip icon on action list items.

**Review by shadowhand — APPROVED** [review]

**Comment by paulfalgout** [comment]
Replaced with https://github.com/RoundingWell/care-ops-frontend/pull/909

**Pull request closed without merging**